image_recognition_openpose/src/image_recognition_openpose/openpose_wrapper.py

```diff
@@ -55,13 +58,18 @@ def _validate_dir(dir_path):
         return dir_path if dir_path[-1] == "/" else dir_path + "/"
 
     def detect_poses(self, image):
-        keypoints, overlayed_image = self._openpose.forward(image, True)
+        datum = self._op.Datum()
```

Comment:
Call `op.Datum()`

Reply:
This is a direct consequence of the way pyopenpose is imported. If that changes, this will change as well.